translationCoreApps/tC_Resources
# Added UGNT v0.2 Titus

## This PR addresses:

Adding the Titus greek UGNT v0.2 files.

## To Test:

Just look and make sure the formatting looks good.
Make sure no data is left out.
Is the manifest good?


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/tc_resources/2)
